packages/charts/src/components/BarChart/BarChart.tsx

```diff
@@ -203,6 +207,7 @@ const BarChart: FC<BarChartProps> = forwardRef((props: BarChartProps, ref: Ref<H
             tickLine={tickLineConfig}
             tickFormatter={primaryMeasure?.formatter}
             height={xAxisHeight}
+            reversed={useIsRTL()}
```

Comment:
please don't call hooks inside props. extract it to `const isRTL = useIsRTL()` somewhere before the return and just use the value here.